python/arcticdb/util/test.py

```diff
@@ -32,6 +32,8 @@
 PANDAS_VERSION = version.parse(pd.__version__)
 CHECK_FREQ_VERSION = version.Version("1.1")
 IS_PANDAS_ZERO = PANDAS_VERSION < version.Version("1.0")
+ARCTICDB_USING_CONDA = os.environ.get("ARCTICDB_USING_CONDA", "0") != "0"
+print(f"ARCTICDB_USING_CONDA {ARCTICDB_USING_CONDA}")
```

Comment:
Please use feature detection instead of resuing this environment variable because we don't want to manually set the latter all the time when running tests locally.

For example, add a boolean variable like `azure_enabled` to Python in `python_modules.cpp`.

Reply:
Fixed